docs/get-started/overview.rst

```diff
@@ -124,3 +124,26 @@ or just cast to a numeric type (e.g. float).::
    # 22530.8
    print(float(unithelper.miles(activity.distance)))
    # 13.9999900581
+
+
+Rate Limits
+===========
+
+Strava imposes rate limits on the usage of its API. This means that the number of
+requests sent to Strava has an upper limit per 15 minutes and per day. These limits
+are not fixed but depend on the client app.
```

Comment:
```suggestion
requests sent to Strava has an upper limit per 15 minutes and per day. Per strava's documentation: 

> The default rate limit allows 100 requests every 15 minutes, with up to 1,000 requests per day.

These limits
are not fixed but depend on the "size" of client app. Strava _may_ chose to adjust rate limits for apps as they grow.
```

Reply:
@jsamoocha i'm playing with this locally and am really confused by the rate limits. i'll add a comment at the end of this review as i may just not understand how this works. I also have a paid account AND my developer app is fairly old so i'm not sure how new rate limits compare to older ones.

Reply:
The 600/15min and 30k/day limits are not the case anymore (although some Strava doc still mentions it). Although "legacy" apps (probably the one you use for testing) still have these default limits. You can check your app's limits through the settings page (https://www.strava.com/settings/api, which isn't available in Strava's mobile app). I'll update the documentation to mention this.